app/api/helpers/mail.py

```diff
@@ -12,7 +12,7 @@
 from app.models.mail import Mail, USER_CONFIRM, NEW_SESSION, USER_CHANGE_EMAIL, SESSION_ACCEPT_REJECT, EVENT_ROLE, \
     AFTER_EVENT, MONTHLY_PAYMENT_EMAIL, MONTHLY_PAYMENT_FOLLOWUP_EMAIL, EVENT_EXPORTED, EVENT_EXPORT_FAIL, \
     EVENT_IMPORTED, EVENT_IMPORT_FAIL, TICKET_PURCHASED_ATTENDEE, TICKET_CANCELLED, TICKET_PURCHASED, USER_EVENT_ROLE, \
-    TEST_MAIL
+    TEST_MAIL, MAIL_TO_EXPIRED_ORDERS
```

Comment:
@uds5501 just like other mail helpers, IMO you should name it according to what you've used everywhere else - `ORDER_EXPIRED` just to keep consistency.

Reply:
@shreyanshdwivedi I was actually just utilising the existing variable. I will change it in my next commit :smile: